controller/api/proxy/client.go

```diff
@@ -5,6 +5,8 @@ import (
 	"google.golang.org/grpc"
 )
 
+// NewClient creates a new gRPC client to the Destination service.
+// TODO: consider moving this into destination-client, or removing altogether.
```

Comment:
👍 Looks like this is only called from the destination-client script, so I'm definitely in favor of removing this file and moving the code directly into the script. But doesn't necessarily have to happen as part of this branch.

Reply:
Sounds good, will address in a subsequent PR.